insights/parsers/messages_compatible.py

```diff
@@ -0,0 +1,80 @@
+"""
+MessagesCompatible - files ``/var/log/messages*`` and systemd journal outputs
+=============================================================================
```

Comment:
Is `MessagesCompatible` descriptive of the implementation for log files that don't have year?  Also, since this is shared it probably makes more sense to include this in `parsers/__init__.py` instead of as a separate module file.